docs: update README.md (#4)
IU-MyMini/InteractiveCourseGrading: README.md

  Interactive Course Grading
</p>
<p align="center">
  <a href="https://my.innopolis.university"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/9c/IU_logo.svg/2560px-IU_logo.svg.png" alt="MyMini"></a>
</p>

<p align="center">
    <em>A simplified, integrated LMS for the Interactive IT Product Dev course, designed to streamline individual and group grade management.</em>
</p>

## Project Goal

> The primary goal of MyMini is to replace the cumbersome Google Sheets-based grade tracking system for the ITPD course with a lightweight, purpose-built Learning Management System (LMS). It aims to provide a clear, secure, and efficient way for students, TAs, and instructors to manage assignments, submissions, and grades—especially for courses that involve both individual and group work.

Reviewer: ```suggestion
The primary goal of MyMini is to replace the cumbersome Google Sheets-based grade tracking system for the ITPD course with a lightweight Learning Management System (LMS). It aims to provide a clear and efficient way for students, TAs, and instructors to manage assignments, submissions, and grades (considering grading for both individual and group work).
```

## Threshold of Success

The project will be considered successful when:
*   TAs and the instructor no longer need to use Google Sheets for managing ITPD grades.
*   Students can reliably view their individual and group grades without seeing the grades of other students.
*   The core workflows — submission upload, peer review assignment, and grading—are fully functional and intuitive for all user roles.

> *   The system is successfully integrated into the `my.innopolis.university` portal using the university's Single Sign-On (SSO).

Reviewer: ```suggestion
*   The system is successfully integrated into the `my.innopolis.university` portal using the university's Single Sign-On.
```

## Description

> MyMini is a focused LMS layer developed specifically for the needs of the Interactive IT Product Dev (ITPD) course at Innopolis University. It directly addresses the pain points of managing complex grading schemes in a shared Google Sheet by providing distinct, role-based views and actions.

Reviewer: ```suggestion
The system will provide a focused LMS layer developed specifically for the needs of the IT Product Dev course at Innopolis University. It directly addresses the pain points of managing complex grading schemes in a shared Google Sheet by providing distinct, role-based views and actions.
```
